.travis.yml

```diff
@@ -20,7 +19,9 @@ script:
   - stack setup --no-terminal
   - stack build --ghc-options=-Werror --no-terminal
   - stack test --ghc-options=-Werror --no-terminal
-  - stack haddock --no-terminal
+  # temporarily use a nightly for haddocks until fix is available in LTS
```

Comment:
Can you link to the haddock issue here so future us-es can keep tabs on when it's resolved (and what the issue is)?

Reply:
Sure. Will do.